core/templates/dev/head/domain/topic/TopicUpdateService.ts

```diff
@@ -448,27 +448,6 @@ angular.module('oppia').factory('TopicUpdateService', [
         });
       },
 
-      /**
-       * Adds an uncategorized skill to a topic and records the change
-       * in the undo/redo service.
-       */
-      addUncategorizedSkill: function(topic, skillSummary) {
```

Comment:
Are we removing this functionality? If yes should we remove this throughout (like from Topic object from backend etc.?)

Reply:
The functionality is still present, just that it doesn't happen in the topic editor. Skills are assigned to topics through the dashboard.